nautobot/circuits/templates/circuits/inc/circuit_termination_fragment.html

```diff
@@ -0,0 +1,74 @@
+{% load helpers %}
```

Comment:
Merge this file back into `circuit_termination.html`, as it's not needed or used now that the circuit termination panel is code-defined.